ntp_node_tree.py

```diff
@@ -1,4 +1,5 @@
 from bpy.types import NodeTree
+import bpy
 
 class NTP_NodeTree:
```

Comment:
Not sure if this type is necessary any longer, at least in the general case. I like what the Geometry Node Tree does with it, but with the `node_tree_vars` dictionary I don't think it's probably necessary for Shader and Compositing nodes.